src/components/dashboard/notifications-tab.yml

```diff
@@ -1,13 +1,28 @@
 notificationsTab:
   info:
-    en: Here you can find your notifications.
-    de: Hier findest du deine Benachrichtigungen.
+    en: Here you can find your notifications about contibutions to your favorite documents, by your favorite users or to documents in rooms that you are a member of.
+    de: Hier findest du deine Benachrichtigungen über Beiträge an deine Lieblingsdokumente, von deine Lieblingsnutzern oder an Dokumente in Räume, in denen du Mitglied bist.
```

Comment:
Hier findest du Benachrichtigungen über Beiträge zu deinen Lieblingsdokumenten, von deine Lieblingsnutzern, oder zu Dokumenten in Räumen, in denen du Mitglied bist.

Maybe we should also scratch the first "your" in the english version?